languages/anaconda3/config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "display_name": "Python (Conda) Environments",
+    "display_name": "Conda Environments - beta",
```

Comment:
Name seems inconsistent with others. Is there a standard here?

Reply:
I keep coming back and forth with the name tbh, i think this is a bit better for the language selection since we could have more "Conda environments" that are not only Python. So something right now it would look like:

```
Conda environments:
 - Python 3.8 (Conda)
 - Python 3.9 (Conda)
 - Julia (Conda)
```

Reply:
Updated to `Conda (Environments)` for consistency and the `(*)` gets removed from the UI.